oiosaml/src/main/java/dk/gov/oio/saml/service/SPMetadataService.java

```diff
@@ -83,23 +83,27 @@ public void createMetadata() throws InternalException, InitializationException {
-		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("dk:gov:saml:attribute:AssuranceLevel", true));
-		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("dk:gov:saml:attribute:SpecVer", true));
-		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("urn:oid:2.5.4.5", true));
-		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("dk:gov:saml:attribute:IsYouthCert", true));
-		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("urn:oid:2.5.29.29", true));
-		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("dk:gov:saml:attribute:PidNumberIdentifier", true));
-		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("dk:gov:saml:attribute:CvrNumberIdentifier", false));
-		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("urn:oid:2.5.4.10", false));
-		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("urn:oid:2.5.4.65", false));
-		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("urn:oid:1.3.6.1.4.1.1466.115.121.1.8", true));
-		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("dk:gov:saml:attribute:RidNumberIdentifier", false));
-		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("dk:gov:saml:attribute:CprNumberIdentifier", true));
-		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("urn:liberty:disco:2006-08:DiscoveryEPR", true));
+		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("https://data.gov.dk/model/core/specVersion", true));
+		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("https://data.gov.dk/model/core/eid/bootstrapToken", false));
+		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("https://data.gov.dk/model/core/eid/privilegesIntermediate", false));
+		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("https://data.gov.dk/concept/core/nsis/loa", true));
+		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("https://data.gov.dk/concept/core/nsis/ial", false));
+		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("https://data.gov.dk/concept/core/nsis/aal", false));
+		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("https://data.gov.dk/model/core/eid/fullName", false));
+		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("https://data.gov.dk/model/core/eid/firstName", false));
+		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("https://data.gov.dk/model/core/eid/lastName", false));
+		attributeConsumingService.getRequestAttributes().add(buildRequiredAttribute("https://data.gov.dk/model/core/eid/email", false));
```

Comment:
Mangler alias attributten ikke?

Reply:
Det diskuteres, NemLog-in vil udleverer de to attributter uanset om tjenesten har ønsket dem eller ej. Det er derfor jeg har undladt dem i metadata.